pybtp/btp.py

```diff
@@ -1545,6 +1547,41 @@ def gattc_read(bd_addr_type, bd_addr, hdl):
     iutctl.btp_socket.send(*GATTC['read'], data=data_ba)
 
 
+def gattc_read_uuid(bd_addr_type, bd_addr, start_hdl, end_hdl, uuid):
+    logging.debug("%s %r %r %r %r %r", gattc_read_uuid.__name__, bd_addr_type,
+                  bd_addr, start_hdl, end_hdl, uuid)
+    iutctl = get_iut()
+
+    gap_wait_for_connection()
+
+    if type(start_hdl) is str:
+        start_hdl = int(start_hdl, 16)
+
+    if type(end_hdl) is str:
+        end_hdl = int(end_hdl, 16)
+
+    data_ba = bytearray()
+
+    bd_addr_ba = addr2btp_ba(bd_addr)
+    start_hdl_ba = struct.pack('H', start_hdl)
+    end_hdl_ba = struct.pack('H', end_hdl)
+
+    if "-" in uuid:
+        uuid = uuid.replace("-", "")
+    if uuid.startswith("0x"):
+        uuid = uuid.replace("0x", "")
+    uuid_ba = binascii.unhexlify(uuid)[::-1]
```

Comment:
Isn't this transformation "uuid_ba = binascii.unhexlify(uuid.translate(None, "-"))[::-1]" used commonly not sufficient here ?